website/events/services.py

```diff
@@ -137,7 +137,9 @@ def cancel_registration(member, event):
 
     if event_permissions(member, event)["cancel_registration"] and registration:
```

Comment:
The events package has the `event_permissions` function in this same file that determines if you should be allowed to do an action like cancelling the registration. So instead of blocking this here you may want to change that permissions function instead.